conf/modules/lidar_lite.xml

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="lidar_lite" dir="lidar">
```

Comment:
The teraranger is not using a laser source, but still based on light (http://www.teraranger.com/technology/). Can we still call that a lidar ?